test/clj_sqs_extended/core_test.clj

```diff
@@ -111,6 +114,61 @@
           (is (contains? stats :stopped-at))))
       (close! handler-chan))))
 
+(deftest handle-queue-terminates-after-restart-count-exceeded
+  (testing "handle-queue terminates when the restart-count exceeds the limit"
+    (let [handler-chan (chan)]
+      (fixtures/with-test-standard-queue
+        (with-redefs-fn {#'sqs/receive-message
+                         (fn [_ _ _] (HttpTimeoutException.
+                                       "Something seems to be wrong with the network."))}
+          #(let [stop-fn (fixtures/with-handle-queue-queue-opts-standard-no-autostop
```

Comment:
what are these `#` for?

Reply:
This is basically what the documentation examples said, I'll have a look later to maybe improve it further.